test/native/helpers.js

```diff
@@ -33,14 +33,9 @@ provideToNativeHtml.mockImplementation( ( html ) => {
 	serializedHtml = html;
 } );
 
-export async function initializeEditor( { initialHtml } ) {
+export async function initializeEditor( props ) {
```

Comment:
This change allows us to pass any Editor prop on the initialization like the capabilities object.